xml/System.Runtime.Loader/AssemblyLoadContext.xml

```diff
@@ -283,10 +369,14 @@
         <Parameter Name="assembly" Type="System.IO.Stream" />
       </Parameters>
       <Docs>
-        <param name="assembly">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="assembly">A byte array that is a COFF-based image containing an emitted assembly.</param>
```

Comment:
I would not use the "emitted" here. How the stream is created is of no concern to this method.